### ISSUE TITLE
rpy2 3.0.0 breaks

### ISSUE THREAD

**Issue by yannikschaelte** [ICB-DCM/pyABC#66]
The latest rpy2 version 3.0.0 seems to have some undesirable changes, which prevent pyabc from working with it out-of-the-box. Depending on the system, occurring errors include ``ValueError: Not an rpy2 R object and unable to cast it into one``, ``module 'rpy2.robjects.pandas2ri' has no attribute 'ri2py'``.

If possible, the current preferred solution is to use rpy2==2.9.5 or 2.9.4, which should work, if that can be installed on the system (also not always the case).

**Comment by yannikschaelte** [comment]
can be solved in rpy2>=3.0.1 via
```python
import rpy2.robjects as robjects
    from rpy2.robjects.conversion import localconverter
    from rpy2.robjects import pandas2ri
```
and then
```python
def r_to_py(object_):
    if isinstance(object_, robjects.DataFrame):
        with localconverter(pandas2ri.converter):
            py_object_ = robjects.conversion.rpy2py(object_)
        return py_object_
    return object_
```
however, also the creation of a rpy2.robjects.ListVector in pyabc.external.dict_to_named_list fails recognizing numpy.float64 values somehow, so something else must be wrong.

**Comment by yannikschaelte** [comment]
fixed via #145
